backend/src/main/java/com/htecgroup/skynest/lambda/converter/LambdaTypeConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.htecgroup.skynest.lambda.converter;
+
+import com.htecgroup.skynest.lambda.LambdaType;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.stream.Stream;
+
+@Converter(autoApply = true)
```

Comment:
those annotations in BucketEntity weren't enough?

Reply:
Nope. We need this so that we can change the enum name if we want, or add new enums wherever we want without messing up the database.

Reply:
we did that manually for the other enums 😔